packages/styletron-react/src/index.js

```diff
@@ -20,7 +20,6 @@ import type {
   ReducerContainer,
   AssignmentCommutativeReducerContainer,
   NonAssignmentCommutativeReducerContainer,
```

Comment:
when looking through yesterday, I noticed the `NonAssignmentCommutativeReducerContainer` type was not used - should it be maintained, or leveraged in this file?

Reply:
Yeah, `NonAssignmentCommutativeReducerContainer` is not used directly so there's no need to import it.

However, the type is inferred in [`composeDynamic`](https://github.com/styletron/styletron/pull/320/commits/5847117a3551043fd3f79a531f5ce429be34adf3#diff-f723eef49306ed376aa6deab2dcb9205L432)(Flow is able to refine the `ReducerContainer` disjoint union via the `assignmentCommutative` property).

Reply:
right, does the type need to be in scope for it to be inferred?